src/stack/Stack.java

```diff
@@ -19,38 +19,49 @@ public Stack(int size) {
 	 * Add element into stack
 	 */
 	public void push(int value) {
-
+		if (tos == data.length - 1) {
+			ensureCapacity();
+		}
+		data[++tos] = value;
 	}
 
 	/**
 	 * Get element from the stack
 	 * @return
 	 */
 	public int pop() {
-
-		return 0;
+		if (isEmpty()) {
+			return -1;
+		} else {
```

Comment:
You don't need **ELSE statement**, just put **return data[tos--];** after **IF statement**.